config/initializers/locale.rb

```diff
@@ -0,0 +1,2 @@
+I18n.load_path += Dir[Rails.root.join('lib', 'locale', '*.{rb,yml}')]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.